src/components/atoms/forms/form-item.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../../sass/mixins";
+@use "../../../sass/settings";
+@use "../../../sass/typography";
+
+.form-item {
+  // @include blg-spacing("bottom", "small", "margin");
+
+  & > :last-child {
+    @include mixins.margin(0, "bottom"); // Margins don't collapse on <select>s
+  }
+}
+
+.form-item__label_container {
+  @supports (display: flex) {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
+.form-item__label {
+  @include mixins.margin(4, "bottom");
+
+  // @include typeg-form-label();
```

Comment:
we can use the font-size-and-vertical-height mixin and just add the bold etc to this